Motion Amendments - June 2019 (#87)
thewca/wca-documents: documents/motions/10.2019.2 - Disciplinary Committee.md

      3. Managing and administrating investigations, reports, decisions, and disciplinary actions.
      4. Keeping all information confidential that needs to stay confidential, based on internal and external guidelines
2. All WCA Staff must fully cooperate with investigations by the WCA Disciplinary Committee.
3. All Registered Speedcubers must fully cooperate with investigations by the WCA Disciplinary Committee.

>    1. All Registered Speedcubers should respond to emails sent on behalf of the WCA Disciplinary Committee in a timely manner.

Reviewer: I feel "a timely manner" is quite vague. My personal opinion is that it would be good to have a definition of what this is (e.g. one week) that can be used. We don't want debates to just turn into an argument about what is/isn't a "timely manner". Although I'm not WDC so I'm happy to be corrected.

Reviewer 2: Well, we can either debate what is a "timely manner" now or later. :P

Author: One week sounds reasonable to me :stuck_out_tongue:

Reviewer 3: It's been a week! ;-)

Reviewer 2: OK, let's go with one week.

Reviewer 4: I agree with 1 week.

Author: I have already included this in a `Staff review` commit that I'll be pushing live later.

>    1. All Registered Speedcubers should respond to emails sent on behalf of the WCA Disciplinary Committee in a one week period.

Reviewer 2: ```suggestion
   1. All Registered Speedcubers should respond to emails sent on behalf of the WCA Disciplinary Committee within one week of each message.
```

Author: I will push a new `Staff review` commit with this right now.

>       1. If a Registered Speedcuber decides to stop responding to the WCA Disciplinary Committee, they must inform the WCA Disciplinary Committee or the WCA Board as to the reasons why.

Reviewer: The phrasing of this sounds odd. If the speedcuber is no longer responding to the WDC, then they would by definition not inform the WDC. Maybe something like:
```suggestion
      1. A Registered Speedcuber must inform the WCA Disiplinary Committee or the WCA Board if they decide to stop responding to the WCA Disciplinary Committee and explain as the the reasons why
```

Reviewer 2: Sure, I think this is a reasonable clarification.

Reviewer 3: `as to the reasons why` seems a bit faux-legalese to me, and it strikes me as more difficult to parse. (Also note that the numbering is wrong in the current PR, but I believe Markdown fixes that automatically.)

What's the main purpose of this? Isn't it a contradiction of `must fully cooperate with investigations` above?
Is the idea that the Registered Speedcuber can decide to stop cooperating, but there will be harsher consequences if they don't communicate that they will do this?

Author: (Where's the numbering wrong?)

I don't think this contradicts the point above as I see it more as a way of enforcing that RS will have the duty to at least inform the WDC that they will be no longer replying to emails instead of just letting them go un-replied.

Reviewer 2: I'd prefer to keep it.

Reviewer 4: I am also in favor of keeping it. Lucas, please respond to Alberto's question.

Author: I have already included the clarification in a `Staff review` commit that I'll be pushing live later, and I double checked but couldn't find where the numbering is wrong.
